python/lsst/ts/athexapod/controller.py

```diff
@@ -247,8 +228,8 @@ async def controller_ready(self):
         """
         ret = await self.write_command("\7", num_line=1)
 
-        comp = ret == chr(177)
-        self.log.debug(f"ret={ret} : {chr(177)} : comp={comp}")
+        comp = ret[0] == chr(177)
```

Comment:
Please extract `chr(177)` to a constant with a meaningful name and use that in this line and the next.